build.gradle

```
@@ -23,16 +23,16 @@ buildscript {
         versionCode = 10002
 
         // SDK and tools
-        compileSdkVersion = 28
+        compileSdkVersion = 29
```

I think we should stick with the recommendations from https://developer.android.com/training/wearables/apps/creating#wear-project and keep this to 28.
```suggestion
        compileSdkVersion = 28
```